assignments/callbacks.js

```diff
@@ -27,12 +27,22 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length);
 }
```

Comment:
so what we are looking here is a bare bones function that will return another function that passes the arr to the callback. And the callback is what is going to determine the length. 

i.e. getLength: 

```
function getLength(arr, cb) {
    return cb(arr);
       //this return is going to return the callback func that has the arr passed as a parameter
}

function length(arr) {
    return arr.length;
}


console.log(getLength(items, length);

```